src/main/java/com/google/codeu/servlets/CinemaLocationServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.codeu.servlets;
```

Comment:
Hi Reine, I think these files do not belong to your current PR. Please branch out from master next time instead of branching out from a feature branch. You might want to do an interactive rebase from master to remove these additional commits.

Reply:
Alright will do, I think somehow I got lost and confused about the git stuff and my intellj was complaining of errors. I didn't know I pushed these additional files because I specifically added only the relevant files. But I'll go find out whats the problem. Thanks for the suggestion